### ISSUE TITLE
2.1.5 beta 2 Disconnect from server

### ISSUE THREAD

**Issue by pljones** [elieserdejesus/JamTaba#1114]
Today I got booted from a server twice, once when alone, once when someone else just joined.

I noticed after the second time I could no longer listen to any other server, either.

Edit: three times, plus disconnected from public chat at some point.  No other local network issues (wired ethernet)

Edit: four

**Comment by pljones** [comment]
Hmm...

It happened today a few times.  In all cases, I was silent (MIDI -> Sampler -> JamTaba, with true silence from the sampler).  So that might be related...

I had to give up and revert to ReaNINJAM in the end.

**Comment by pljones** [comment]
Used beta3 today and didn't get issues -- though I didn't really have similar test scenarios.

**Comment by elieserdejesus** [comment]
I'm using the beta (VST plugin) without issues here. No disconnection problems.

@DoublebassOO mentioned jtba is crashing after some minutes when using video. But I tested this scenario and is working fine in my side. No clues about the issue at moment.

**Comment by DoublebassOO** [comment]
i havent the beta3 intalled.    but tonight   with it no more problem with the video even more than an hour  !!  coool
this beta seems now ok on my side too !!  Bravo!!

**Comment by pljones** [comment]
I was having a lot of disconnects with beta3 last week and had to revert to ReaNINJAM again (not helped by the levels issue in #1121).

**Comment by pljones** [comment]
OK, after another week, I _think_ it's related to silence.  Not sure but if I don't transmit anything for an interval, I think I get kicked by the client.  (It doesn't happen in ReaNINJAM, so it's not the server.)

(Also getting _lots_ of disconnects on the public chat server.)

**Comment by elieserdejesus** [comment]
> OK, after another week, I think it's related to silence. Not sure but if I don't transmit anything for an interval, I think I get kicked by the client. (It doesn't happen in ReaNINJAM, so it's not the server.)

I tested this possibility. My tests was:

1 - Open Jamtaba Standalone
2 - Turn off xmit (using one channel) before enter in the server
3 - Connect in the server
4 - Wait a long time (> 30 minutes)

I tested in my local server, ninbot and ninjamer. **No problem in the 3 tests, no disconnections**.

Maybe the issue is triggered in some very specific conditions, not easy to test and detect the bug.
